WcaOnRails/app/models/team.rb

```diff
@@ -21,6 +21,16 @@ def membership_periods_cannot_overlap_for_single_user
     end
   end
 
+  BOARD_FRIENDLY_ID = 'board'
```

Comment:
Loving this! Thanks for taking care of this cleanup. We're not super consistent about this, but I think it's nice if constants like this come at the top of a class definition.

Reply:
Ah, I probably copied from competitions.rb, in which constants are after the validations.
I can change this one.